src/components/content-tab-snapshots.tsx

```diff
@@ -182,16 +193,22 @@ function SnapshotRow( {
 				) : (
 					<>
 						<Tooltip
-							disabled={ ! isOffline }
-							icon={ offlineIcon }
-							text={ updateDemoSiteOfflineMessage }
+							disabled={ ! ( isOffline || snapshotCreationBlocked ) }
+							{ ...( isOffline && { icon: offlineIcon } ) }
+							text={ isOffline ? updateDemoSiteOfflineMessage : blockedUserMessage }
 						>
 							<Button
-								aria-description={ isOffline ? updateDemoSiteOfflineMessage : '' }
-								aria-disabled={ isOffline }
+								aria-description={
+									snapshotCreationBlocked
+										? blockedUserMessage
+										: isOffline
+										? updateDemoSiteOfflineMessage
+										: ''
+								}
+								aria-disabled={ isOffline || snapshotCreationBlocked }
 								variant="primary"
 								onClick={ () => {
-									if ( isOffline ) {
+									if ( isOffline || snapshotCreationBlocked ) {
```

Comment:
In my suggested PR I'm moving this condition into a new variable, because it's repeated multiple times.

Reply:
Makes sense, thanks. I have merged that PR so I will keep your approach 👍